recipe/meta.yaml

```diff
@@ -19,6 +23,7 @@ requirements:
     - setuptools
   run:
     - python
+    - certifi
 
 test:
```

Comment:
Could we add some sort of test to make sure `certifi` is picked up and used? If you would like, please feel free to pilfer from [this]( https://github.com/conda-forge/apache-libcloud-feedstock/pull/1/files#diff-e178b687b10a71a3348107ae3154e44cR34 ) and tweak it. Would be nice to have it assert that it picked up `certifi`'s certs in particular.